rllib/tuned_examples/bc/benchmark_atari_pong_bc.py

```diff
@@ -145,31 +146,32 @@ def _env_creator(cfg):
 # There are many run numbers, we choose the first one for demonstration. This
 # can be chosen by users. To use all data use a list of file paths (see
 # `num_shards`) and its usage further below.
-run_number = 1
-# num_shards = 1
+# run_number = 1
+# # num_shards = 1
 
-# Make the temporary directory for the downloaded data.
-tmp_path = "/tmp/atari"
-Path(tmp_path).joinpath(game).mkdir(exist_ok=True, parents=True)
-destination_file_name = f"{tmp_path}/{game}/run_{run_number}-00000-of-00001"
+# # Make the temporary directory for the downloaded data.
+# tmp_path = "/tmp/atari"
+# Path(tmp_path).joinpath(game).mkdir(exist_ok=True, parents=True)
+# destination_file_name = f"{tmp_path}/{game}/run_{run_number}-00000-of-00001"
 
-# If the file is not downloaded, yet, download it here.
-if not Path(destination_file_name).exists():
-    # Define the bucket and source file.
-    bucket_name = "rl_unplugged"
-    source_blob_name = f"atari/{game}/run_{run_number}-00000-of-00100"
+# # If the file is not downloaded, yet, download it here.
+# if not Path(destination_file_name).exists():
+#     # Define the bucket and source file.
+#     bucket_name = "rl_unplugged"
+#     source_blob_name = f"atari/{game}/run_{run_number}-00000-of-00100"
 
-    # Download the data from the bucket.
-    storage_client = storage.Client.create_anonymous_client()
-    bucket = storage_client.bucket(bucket_name)
-    blob = bucket.blob(source_blob_name)
-    blob.download_to_filename(destination_file_name)
+#     # Download the data from the bucket.
+#     storage_client = storage.Client.create_anonymous_client()
+#     bucket = storage_client.bucket(bucket_name)
+#     blob = bucket.blob(source_blob_name)
+#     blob.download_to_filename(destination_file_name)
 
 # Define the config for Behavior Cloning.
 config = (
     BCConfig()
     .environment(
         env="WrappedALE/Pong-v5",
+        # TODO (sven): Does this have any influence in connectors?
```

Comment:
Great point! You are right and this setting is NOT propagated to the connectors. Not relevant for Pong as its rewards are all 1 anyways, but for other Atari benchmarks, this could matter.

Reply:
Thanks for the clarification. There is actually another one: 

```python
        # TODO (sven): Has this any influence in the connectors?
        actions_in_input_normalized=True,
```
Does this have an influence - or should it? It is not recorgnized, yet, in the offline API.